<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 191, 188);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>1101</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="shockButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 30px;
height: 40px;
width: 40px;
background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>320</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="LED4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
height: 20px;
width: 20px;
background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="analyLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic4.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>121</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="LED1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
height: 20px;
width: 20px;
background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cprLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Display">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(87, 227, 137);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="shocks">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Shocks: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="ecg">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="timer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>320</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="LED5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
height: 20px;
width: 20px;
background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="padsLabel_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic5_2.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="padsLabel_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic5_1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="LED2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
height: 20px;
width: 20px;
background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="emergLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic2.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>430</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>120</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="padsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/pic3.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LED3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
height: 20px;
width: 20px;
background-color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="end">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>460</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>272</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkboxSlowCompression">
        <property name="text">
         <string>Slow Compression</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkboxPlacedPads">
        <property name="text">
         <string>Placed Pads</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxWeakeCompression">
        <property name="text">
         <string>Weak Compression</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Battery</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Patient state</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" rowspan="2" colspan="2">
       <widget class="QTextEdit" name="logs"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxPatientState">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Unshockable</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>V Fib</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>V Tach</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Logs</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="sliderBattery">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnPreformCpr">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>350</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Perform cpr</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
